dags/longitudinal.py

```diff
@@ -40,6 +40,17 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/addon_recommender.sh",
     dag=dag)
 
+ensemble_recommender = EMRSparkOperator(
+    task_id="ensemble_recommender",
+    job_name="Extract the addon data for each Telemetry user to DynamoDB",
```

Comment:
This parameter is actually what the EMR cluster will be named -- it'd actually be kind of tough to tie that back to the correct airflow job with this name. Please pick a name that's closer to the task_id.